design-documents/graph-ql/coverage/SubscribeEmailToNewsletter.graphqls

```diff
@@ -0,0 +1,7 @@
+type Mutation {
+    subscribeEmailToNewsletter(email: String!): SubscribeOutput @doc(description:"Adds an email into a newsletter subscription")
```

Comment:
To be consistent with other mutations, please change output to `SubscribeEmailToNewsletterOutput` (mutation name + suffix)

Reply:
**done**

Thank you!

Reply:
@paliarush , did you have a chance to review the applied changes?

Thank you!